lib/seattleflu/id3c/api/routes.py

```diff
@@ -14,12 +14,21 @@
 blueprints.append(api_v3)
 
 @api_v1.route("/shipping/return-results/<barcode>", methods = ['GET'])
+def get_barcode_results_v1(barcode=None):
+    """
+    Mark old endpoint as 410 Gone
+    """
+    return "use /v2/shipping/return-results/<barcode>", 410
+
+
+@api_v2.route("/shipping/return-results/<barcode>", methods = ['GET'])
 @cross_origin(origins=[
     "https://seattleflu.org",
     "https://dev.seattleflu.org",
+    "http://localhost:3000",
     "http://localhost:8080"])
```

Comment:
By "moving" the `@cross_origin` decorator from v1 → v2 instead of copying it, the v1 endpoint won't actually ever get a chance to respond 410 Gone to a cross-origin request from the website frontend. It'll be blocked by CORS first.

This doesn't really matter, as the end result is similar (the request still fails), but I wanted to point out this nuance as I don't think it's what you intended.

Reply:
That makes sense. I'll duplicate the `@cross_origin` decorator across v1 and v2.